ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -43,7 +44,7 @@
  * Mobile Ads SDK mediation.
  */
 @Keep
-public class VungleInterstitialAdapter
+public class VungleInterstitialAdapter extends VungleBaseAdapter
```

Comment:
@HuFangshuai 
Question here for Admob from last night. If we are to deprecate MediationInterstitialAdapter, MediationBannerAdapter, what replaces MediationInterstitialAdapter.showInterstitial and MediationBannerAdapter.getBannerView()?

Reply:
MediationInterstitialAd and MediationBannerAd.